hpctestlib/system/ssh/host_keys.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016-2022 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import time
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+import reframe.utility.typecheck as typ
+
+
+@rfm.simple_test
+class host_keys_check(rfm.RunOnlyRegressionTest):
+    '''host keys age check
+
+    The host keys should be renewed regularly.
+    In this case, we are checking against the
+    max_key_age variable
+    '''
+
+    #: Parameter list with all host keys to check
+    #:
+    #: The test skips if a key is not found
+    #:
+    #: :type: :class:`str`
+    #: :values: ``['/etc/ssh/ssh_host_rsa_key',
+    #:              '/etc/ssh/ssh_host_ecdsa_key',
+    #:              '/etc/ssh/ssh_host_ed25519_key']``
+    host_keys = parameter([
```

Comment:
What's the reason you prefer a parameter here instead of a variable with a default value for the keys? Using a variable would make it easier atm to change the list of keys to check from the command line.

PS: I'm not against the parameters, but trying to understand as both solutions have their merits.

Reply:
The `sn.all` that you commented on above was a leftover of the test when I used a variable instead of the parameter. I only used a parameter here because the test became a tiny simpler compared to the variable version.
I can have it as a variable too. I don't mind.